src/modules/Common/components/Column.tsx

```diff
@@ -39,10 +39,13 @@ const Column: React.FC<ColumnProps> = ({
       )}
       {...props}
     >
-      <div className={s.column_header}>
-        {title && <h2 className={s.column_title}>{title}</h2>}
-        {subtitle && <h3 className={classNames(s.column_subtitle, 'h3__light')}>{subtitle}</h3>}
-      </div>
+      {title ||
```

Comment:
this seems weird `{(title || subtitle) && `

Reply:
This avoids the creation of `<div className={s.column_header}>` if `title`and `subtitle` are empty.